### ISSUE TITLE
Safety run failed

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/karmabot#108]
https://github.com/PyBites-Open-Source/karmabot/actions/runs/4073711282

@pogross maybe time to upgrade some of the packages ...

**Comment by pogross** [comment]
Updates on my fork are already done. 
Will also update SQLAlchemy calls for the new 2.0 version in parallel

**Comment by bbelderbos** [comment]
Sweet! Thanks Patrick, so we can expect one big PR for this?

**Comment by pogross** [comment]
Yup, hopefully this week

**Comment by pogross** [comment]
fixed
